scripts/xe-backup-metadata

```diff
@@ -48,6 +48,29 @@ function usage {
     exit 1
 }
 
+function uuid5 {
```

Comment:
missing hash of original commit in the commit message:  084c6a9e940e578a66f0b2fa34efd5f5ed61b989